hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/token/Hip17UnhappyTokensSuite.java

```diff
@@ -314,6 +322,7 @@ private HapiSpec cannotUpdateNftFeeScheduleWhenDeleted() {
                 .given(
                         newKeyNamed(ADMIN_KEY),
                         newKeyNamed(FEE_SCHEDULE_KEY),
+                        newKeyNamed(SUPPLY_KEY),
```

Comment:
Is this change necessary? How was it passing before on mono-service but not now without it? Is it a bug in mono service?

Reply:
Yes, not sure how it was passing in the mono service. We are setting supply key to the token below, but if we don't create it here on line 325 it fails with Missing 'supply key', which makes sense.